app/models/schema/plum/local.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+module Schema
+  module Plum
+    ##
+    # Defines the attributes local to Plum
+    module Local
```

Comment:
Same as above: breaking these out by source would be nice, since some of these properties are our own local properties, but some (alternative, identifier, etc.) are DC.